pkg/cni/plugin/plugin.go

```diff
@@ -106,24 +104,24 @@ func getPrevCniResult(conf *cniConf) (*cniv1.Result, error) {
 	return cniv1PrevResult, nil
 }
 
-func enableXdpAuth(ifname string) error {
```

Comment:
we should still keep it in cni like manage

Reply:
done